### ISSUE TITLE
ContextMenu does not fire close event

### ISSUE THREAD

**Issue by valekask** [isaacplmann/ngx-contextmenu#83]
Recently I've updated context-menu plugin to the latest version, and now `close` event is not firing.

my package.json
```
"@angular/...": "^5.0.0",
"@angular/cdk": "2.0.0-beta.12",
"ngx-contextmenu": "^4.1.1"
```

Please note that context-menu component in a different component from my list, so I work with context-menu via ContextMenuService as described on https://github.com/isaacplmann/ngx-contextmenu#context-menu-in-a-different-component

**Comment by isaacplmann** [comment]
You're right.  This is a bug I need to fix.

**Comment by ryanstelmat** [comment]
I'm seeing this as well

**Comment by isaacplmann** [comment]
This should be fixed in 4.1.2.
